src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
 
+        Machine excavator = new Excavator();
+        excavator.setName("excavator");
+
+        Machine bulldozer = new Bulldozer();
+        bulldozer.setName("bulldozer");
+
+        Machine truck = new Truck();
+        truck.setName("truck");
+
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
+
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
Write informative messages in methods. The `doWork` method should indicate what type of `Machine` is working. Ensure that the `doWork` method in each `Machine` subclass prints an informative message about the machine type.